test/test_helper.rb

```diff
@@ -13,6 +13,10 @@ def self.mock_rest_client=(mock_client)
     @mock_rest_client = mock_client
   end
 
+  class << self
+    remove_method :execute_request
+  end
```

Comment:
This is to fix a warning caused by `execute_request` being redefined at the class level.